packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -46,24 +46,29 @@ const conf = module.exports = convict({
   basket: {
     api_key: {
       default: 'test key please change',
-      doc: 'Basket API key',
+      doc: 'DEPRECATED - Basket API key',
       format: String
     },
     api_timeout: {
       default: '5 seconds',
-      doc: 'Timeout for talking to the Basket API server, in ms',
+      doc: 'DEPRECATED - Timeout for talking to the Basket API server, in ms',
       format: Number
     },
     api_url: {
       default: 'http://127.0.0.1:10140',
-      doc: 'Url for the Basket API server',
+      doc: 'DEPRECATED - Url for the Basket API server',
+      format: String
+    },
+    email_preferences_url: {
+      default: 'https://basket.allizom.org/fxa/',
+      doc: 'DEPRECATED - Url for the Basket proxy server',
```

Comment:
@shane-tomlinson this doc string isn't right. And the default URL is for basket staging. This is known right? The prod URL will be different? Perhaps the default should be basket prod?

Reply:
You are right, and this was intentional. We kinda/sorta go back and forth about whether to put production values as default, or development values. We are settling on development values. I'll open a PR against the cloudops-deployment repo with the correct URL for prod.